app/[locale]/email/verification-failed/page.tsx

```diff
@@ -1,20 +1,28 @@
 'use client'
 
 import React from 'react';
-import PageWrapper from '@/components/PageWrapper';
-import ContentContainer from '@/components/ContentContainer';
-
+import PageHeader from '@/components/PageHeader';
+import {
+  LayoutWithPanel,
+  ContentContainer,
+} from '@/components/Container';
 
 const VerificationFailed: React.FC = () => {
   return (
-    <PageWrapper title={'Verification failed'}>
-      <h1>Verification failed</h1>
-      <ContentContainer>
-        <div className="container">
-          <p>You&lsquo;re email verification failed.</p>
-        </div>
-      </ContentContainer>
-    </PageWrapper >
+    <>
+      <PageHeader title="Verification failed" />
+      <LayoutWithPanel>
```

Comment:
Change this to `LayoutContainer`. Layoutwithpanel shouldn't be used without either a drawercontainer or sidebarcontainer.

Reply:
This is most likely why you had issues with the incorrect width.